core/src/mindustry/client/Client.kt

```diff
@@ -253,6 +253,18 @@ object Client {
             sendMessage("!" + if (args.size == 1) args[0] else "")
         }
 
+        register("tl [lang] [message...]", Core.bundle.get("client.command.tl.description")) { args, _ ->
+            if (args.size == 0) return@register
+            val msg = if (args.size == 1) args[0]
+                      else if (supportedLangs.contains(args[0])) args[1]
+                      else args[0] + " " + args[1]
```

Comment:
This translates the language and the actual content of the message if both are specified?

Reply:
Only if arg[0] isn’t a language code